src/input_common/CMakeLists.txt

```
@@ -30,3 +34,4 @@ endif()
 
 create_target_directory_groups(input_common)
 target_link_libraries(input_common PUBLIC core PRIVATE common ${Boost_LIBRARIES})
+target_link_libraries(input_common PUBLIC ${LIBUSB_LIBRARIES})
```

```suggestion
target_include_directories(input_common PRIVATE ${LIBUSB_INCLUDE_DIR})
target_link_libraries(input_common PUBLIC ${LIBUSB_LIBRARIES})
```
Need to tell CMake where to look for `libusb.h`. Fixes https://github.com/citra-emu/citra/pull/5735#issuecomment-826104156